ui_framework/src/components/header/_header.scss

```diff
@@ -0,0 +1,34 @@
+// Header. Includes breadcrumbs of nav buttons.
+
+@include component('kuiHeader') {
+  background: $kuiHeaderBackgroundColor;
+  border-bottom: $kuiBorderThin;
+
+  @include child('logo') {
+    position: relative;
+    height: $kuiHeaderChildSize;
+    padding: $kuiSize $kuiSizeXL;
+    display: inline-block;
+    vertical-align: middle;
+
+    @include kuiHeaderSeparation('right');
```

Comment:
If we change the `kuiHeaderItem` to put the separator on the right, then we can remove this mixin here and wrap the logo inside of the kuiHeaderItem instead.